app/models/prayer.rb

```diff
@@ -0,0 +1,7 @@
+class Prayer < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :resident
+  belongs_to :route, optional: true
+
```

Comment:
I think we're missing one piece, a Prayer may have one commitment through a Route.